### PR TITLE
feat: Merge models passed from context and props

### PR DESCRIPTION
**Pull request by spencer-brown**
Fixes https://github.com/mongodb-js/connect-backbone-to-react/issues/14.

### CODE REVIEW

File: test/backbone-provider.test.js [hunk 1]
```diff
@@ -106,4 +106,94 @@ describe('BackboneProvider', function() {
...
+          <ConnectedPassingParent />
+        </BackboneProvider>
+      );
+
+      const modelsFromContext = wrapper
+        .find('.name')
+        .findWhere((n) => n.text() === userModel.get('name'))
+        .length;
+      const modelsFromParent = wrapper
+        .find('.name')
```
Comment (hswolff) [inline test/backbone-provider.test.js]: Curious if we should put `<ConnectedChild />` within a div with a className so we can scope this lookup to just the child to be more explicit about where the props value is coming from?

**Review by hswolff — CHANGES_REQUESTED** [review]
One small change then should be good to go.

**Comment by hswolff** [comment]
Bump on this, would love to get this merged.

**Comment by ghmeier** [comment]
Hey @hswolff, just made the quick change adding a class to the wrapper div since @spencer-brown has been busy. Hyped to get this feature added.

**Comment by hswolff** [comment]
Thanks @ghmeier !

**Comment by hswolff** [comment]
Published as `v1.5.0`

**Pull request merged**